cmd/zrok/sharePublic.go

```diff
@@ -11,23 +17,22 @@ import (
 	"github.com/openziti/zrok/tui"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
```

Comment:
still reformatting the imports :-(

Reply:
Weird, the other file with import changes formatted themselves differently. 🙃 
I think the newline is the culprit, will fix